### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by Martinspire** [leny/atom-inc-dec-value#3]
I'm getting this deprecation error:

Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (C:\Users\Martin\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:808:9)
Package.hasActivationCommands (C:\Users\Martin\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:733:20)
<unknown> (C:\Users\Martin\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:185:24)
Package.measure (C:\Users\Martin\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:163:15)
Package.load (C:\Users\Martin\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:177:12)
PackageManager.loadPackage (C:\Users\Martin\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package-manager.js:355:14)
```

This is going to be a problem when 1.0 comes out but i think its an easy fix?

**Comment by leny** [comment]
Fixed in `0.0.7`. Thanks for the report.

**Comment by Martinspire** [comment]
Tnx for fixing ;)
